spec/vectorsearch/weaviate_spec.rb

```diff
@@ -55,11 +70,25 @@
       ).to receive(:get)
         .with(
           class_name: "products",
-          near_text: "{ concepts: [\"earth\"] }",
+          near_vector: "{ vector: [-0.0018150936, 0.0017554426, -0.022715086] }",
           limit: "4",
           fields: "content _additional { id }"
         )
         .and_return(fixture)
+
+      allow_any_instance_of(
+        ::OpenAI::Client
+      ).to receive(:embeddings).and_return({
+        "data" => [
+          {
+            "embedding" => [
+              -0.0018150936,
+              0.0017554426,
+              -0.022715086
+            ]
+          }
+        ]
+      })
     end
 
```

Comment:
Also I would really appreciate if you can write the spec for the `#ask` method. Thank you!

Reply:
OK. I wrote!